pkg/kubelet/cm/dra/state/checkpoint.go

```diff
@@ -18,51 +18,88 @@ package state
 
 import (
 	"encoding/json"
+	"hash/crc32"
 
-	"k8s.io/kubernetes/pkg/kubelet/checkpointmanager"
-	"k8s.io/kubernetes/pkg/kubelet/checkpointmanager/checksum"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/kubernetes/pkg/kubelet/checkpointmanager/errors"
 )
 
-var _ checkpointmanager.Checkpoint = &DRAManagerCheckpoint{}
+const (
+	CheckpointAPIGroup   = "checkpoint.dra.kubelet.k8s.io"
+	CheckpointKind       = "DRACheckpoint"
+	CheckpointAPIVersion = CheckpointAPIGroup + "/" + Version
```

Comment:
I think we should get rid of the `Version` variable now and just hard-code this as `/v1` now that we have refactored to a single package

Reply:
done